lib/web_ui/lib/src/engine/semantics/semantics_helper.dart

```diff
@@ -189,12 +189,12 @@ class DesktopSemanticsEnabler extends SemanticsEnabler {
   }
 
   @override
-  html.Element prepareAccesibilityPlaceholder() {
-    _semanticsPlaceholder = html.Element.tag('flt-semantics-placeholder');
+  html.Element? prepareAccesibilityPlaceholder() {
```

Comment:
this is never null (related to the previous comment)

Reply:
Done.